src/algebra/module/localized_module.lean

```diff
@@ -0,0 +1,358 @@
+/-
+Copyright (c) 2022 Jujian Zhang. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Jujian Zhang
+-/
+
+import group_theory.monoid_localization
+import ring_theory.localization.basic
+
+/-!
+# Localized Module
+
+Given a commutative ring `R`, a multiplicative subset `S ⊆ R` and an `R`-module `M`, we can localize
+`M` by `S`. This gives us a `localization S`-module.
+
+## Main definitions
+
+* `localized_module.r` : the equivalence relation defining this localization, namely
+  `(m, s) ≈ (m', s')` if and only if if there is some `u : S` such that `u • s' • m = u • s • m'`.
+* `localized_module M S` : the localized module by `S`.
```

Comment:
My feeling would be that `localized_module S M` would be a more sensible order

Reply:
I swapped the order